main/app-menu.js

```diff
@@ -0,0 +1,56 @@
+const { Menu, MenuItem, ipcMain } = require('electron')
+const { ipcMainEvents } = require('./ipc')
+
+function setupAppMenu (/** @type {import('./typings').Context} */ ctx) {
+  if (process.platform !== 'darwin') return
+
+  // Add "Check for updates..." item to the Application menu on MacOS
+  const menu = Menu.getApplicationMenu()
+  if (!menu) return
+
+  menu.items[0].submenu?.insert(1, new MenuItem({
+    id: 'checkForUpdates',
+    label: 'Check For Updates...',
+    click: () => { ctx.manualCheckForUpdates() }
+  }))
+  menu.items[0].submenu?.insert(2, new MenuItem({
+    id: 'checkingForUpdates',
+    label: 'Checking For Updates',
+    enabled: false,
+    visible: false
+  }))
```

Comment:
I am not sure where other Electron applications place "Check for Updates..." item on Windows & Linux, so I am adding this feature to macOS only to keep things simple.

If you think we should add this item to the application menu on other platforms too, then let's create follow-up GH issues, one per platform.

Reply:
I think we should add these items there too 👍 

https://github.com/filecoin-project/filecoin-station/issues/63
https://github.com/filecoin-project/filecoin-station/issues/64

Reply:
Thanks for creating the GH issues, @juliangruber 🙇🏻 I'll mention them in a code comment.